src/App.js

```diff
@@ -1,12 +1,54 @@
-import React from 'react';
+import React, {
+    useEffect,
+    useState
+} from 'react';
 import './App.css';
+import axios
+    from "axios";
+import PokemonCard from './components/PokemonCard'
+import logo from './assets/logo.jpg';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+    const [pokemon, setPokemon] = useState([]);
+    const [endpoint, setEndpoint] = useState('https://pokeapi.co/api/v2/pokemon');
+
+    useEffect(() => {
+        async function fetchPokemon() {
+            try {
+                const result = await axios.get(`${endpoint}`)
+                console.log(result.data);
+                setPokemon(result.data)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchPokemon();
+    }, [endpoint]);
+
+    return (
+      <>
+      <body>
+      <img className="logo" src={logo} alt="pokemon"/>
+         <div>
+             <button
+                 type="button"
+                 onClick={() => setEndpoint(pokemon.previous)}
+             >previous</button>
+             <button
+                 type="button"
+                 onClick={() => setEndpoint(pokemon.next)}
+             >next</button>
+         </div>
+         <div className="all-cards">
+             {pokemon.results && pokemon.results.map((poke) => {
```

Comment:
Hier kan de check dan wat simpeler:
```suggestion
             {pokemon && pokemon.results.map((poke) => {
```